.github/workflows/chromatic.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
         uses: peter-evans/find-comment@v1
         id: latestComment
         with:
-          issue-number: ${{ github.event.issue.pull_request }}
+          issue-number: ${{ github.event.issue.number }}
```

Comment:
Yep this feels correct for this particular line. I'm struggling to wrap my head around the rest of the workflow to understand if other changes are also needed, but lets merge this and see if it fixes the issue.